.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+      - id: check-yaml
```

```suggestion
      - id: check-yaml
      - id: mixed-line-ending
        args: ["--fix=lf"]
```
for windows users (like me 😉)